backend/apps/issue_routing/services/router.py

```
@@ -0,0 +1,101 @@
+            issue_number=issue_number,
+            issue_title=title,
+            issue_body=body,
+            detected_domains=detected_domains,
+            suggested_maintainers=matches,
+            routing_score=primary_match['score'] if primary_match else 0,
+            confidence=primary_match['score'] if primary_match else 0,
+            status='routed'
+        )
+
+        if primary_match:
+            primary_user = User.objects.get(id=primary_match['user_id'])
```

Ruff (0.15.21)

[error] Undefined name `User`

(F821)